### DISCUSSION TITLE
Mirth Rest API: Interacting and Displaying data

### DISCUSSION THREAD

**Discussion by rjkroll** [nextgenhealthcare/connect#5252] [Q&A]
Hi,

This is more of a general question, not really a support question, but I was looking to do a side-project at work involving hitting Mirth's API and displaying the results and learning some new technology along the way.   

Essentially, I just want to create an interactive webpage, or small app, that just hits the Mirth API, gets a channel listing of deployed channels, and lists them, while also pulling in and displaying some data from the other available API endpoints.

I am familiar with working with API's from within Mirth itself, and hitting them with CURL, etc, but I was wondering what everyone else would use to build this out?    If you were looking to do the same thing, what front-end/back-end would you use?   Would you create a webpage or an actual app?   HTML and Javascript?  Python and Django?  

Thanks in advance,

**Comment by siddharthsharma9258** [comment]
You could use nodeJS or express as a backend service to call the Mirth APIs, and react for the front end. React is popular right now.
This would be easy to spin up.
